recipes/ndtypes/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install || exit 1
+copy python\*.py "%RECIPE_DIR%"
```

Comment:
What's this `copy` being used for?

Reply:
As I understood, files need to be in the RECIPE_DIR, otherwise they aren't present in the test phase.  I pretty much hit this problem:

https://groups.google.com/a/continuum.io/forum/#!msg/anaconda/8ZNwIQeGuNg/AzAGd90hBwAJ